Osquery: Update exported fields reference for osquery 5.5.1 (#143754)
elastic/kibana: docs/osquery/exported-fields-reference.asciidoc

* _systemd_units.active_state_ - The high-level unit activation state, i.e. generalization of SUB

*activity* - keyword, number.long

> * _unified_log.activity_ - the activity ID associate with the entry

Reviewer: Some descriptions have periods at the end and some don't. In a future iteration, it might be good to set a standard for this so that all descriptions are punctuated the same. :)

*comment* - keyword, text.text

* _authorizations.comment_ - Label top-level key
* _authorized_keys.comment_ - Optional comment
* _docker_image_history.comment_ - Instruction comment
* _etc_protocols.comment_ - Comment with protocol description

> * _etc_services.comment_ - Optional comment for a service.

Reviewer: I'd recommend removing if possible so it follows the same punctuation convention that the other list items are using.

*hostname* - keyword, text.text

> * _curl_certificate.hostname_ - Hostname to CURL (domain[:port], e.g. osquery.io)

Reviewer: ```suggestion
* _curl_certificate.hostname_ - Hostname to CURL (domain[:port], for example, osquery.io)
```

Reviewer: We usually try to avoid using abbreviations since these are not always understsood by nonnative-English-speaking folks.

* _lxd_cluster_members.message_ - Message from the node (Online/Offline)
* _selinux_events.message_ - Message
* _syslog_events.message_ - The syslog message

> * _unified_log.message_ - composed message

Reviewer: ```suggestion
* _unified_log.message_ - Composed message
```

> * _services.pid_ - the Process ID of the service

Reviewer: ```suggestion
* _services.pid_ - The Process ID of the service
```

* _shared_memory.pid_ - Process ID to last use the segment
* _socket_events.pid_ - Process (or thread) ID

> * _unified_log.pid_ - the pid of the process that made the entry

Reviewer: ```suggestion
* _unified_log.pid_ - The pid of the process that made the entry
```

*process* - keyword, text.text

* _alf_explicit_auths.process_ - Process name explicitly allowed

> * _unified_log.process_ - the name of the process that made the entry

Reviewer: ```suggestion
* _unified_log.process_ - The name of the process that made the entry
```

*sender* - keyword, text.text

* _asl.sender_ - Sender's identification string.  Default is process name.

> * _unified_log.sender_ - the name of the binary image that made the entry

Reviewer: ```suggestion
* _unified_log.sender_ - The name of the binary image that made the entry
```

* _kva_speculative_info.stibp_support_enabled_ - Windows uses STIBP.

*storage* - keyword, number.long

> * _unified_log.storage_ - the storage category for the entry

Reviewer: ```suggestion
* _unified_log.storage_ - The storage category for the entry.
```

*subsystem* - keyword, text.text

* _system_controls.subsystem_ - Subsystem ID, control type

> * _unified_log.subsystem_ - the subsystem of the os_log_t used

Reviewer: ```suggestion
* _unified_log.subsystem_ - The subsystem of the os_log_t used
```

* _bpf_process_events.tid_ - Thread ID
* _bpf_socket_events.tid_ - Thread ID

> * _unified_log.tid_ - the tid of the thread that made the entry

Reviewer: ```suggestion
* _unified_log.tid_ - The tid of the thread that made the entry
```

Reviewer: Is the "tid" the Thread ID?

*timestamp* - keyword, text.text

* _time.timestamp_ - Current timestamp (log format) in UTC

> * _unified_log.timestamp_ - unix timestamp associated with the entry

Reviewer: ```suggestion
* _unified_log.timestamp_ - Unix timestamp associated with the entry
```
